### PR TITLE
Update install config to match v1beta4 schema.

### PR DESCRIPTION
**Pull request by squeed**
This renames some network fields, but doesn't change behavior.

### CODE REVIEW

File: ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml [hunk 1]
```diff
@@ -221,18 +221,19 @@ objects:
...
+          clusterNetwork:
           - cidr:             10.128.0.0/14
-            hostSubnetLength: 9
+            hostPrefix: 23
           machineCIDR: 10.0.0.0/16
-          serviceCIDR: 172.30.0.0/16
-          type:        OpenShiftSDN
+          serviceNetwork:
+          - 172.30.0.0/16
+          networkType:        OpenShiftSDN
```
Comment (staebler) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml]: Trim whitespace.

File: ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml [hunk 2]
```diff
@@ -245,17 +246,18 @@ objects:
...
-          clusterNetworks:
+          clusterNetwork:
           - cidr:             10.128.0.0/14
-            hostSubnetLength: 9
+            hostPrefix: 23
           machineCIDR: 10.0.0.0/16
-          serviceCIDR: 172.30.0.0/16
+          serviceNetwork:
+          - 172.30.0.0/16
           type:        OpenShiftSDN
```
Comment (staebler) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml]: s/type/networkType/

Trim whitespace.

File: ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml [hunk 3]
```diff
@@ -266,17 +267,18 @@ objects:
...
-          clusterNetworks:
+          clusterNetwork:
           - cidr:             10.128.0.0/14
-            hostSubnetLength: 9
+            hostPrefix: 23
           machineCIDR: 10.0.0.0/16
-          serviceCIDR: 172.30.0.0/16
+          serviceNetwork:
+          - 172.30.0.0/16
           type:        OpenShiftSDN
```
Comment (staebler) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: s/type/networkType/

Trim whitespace.

File: ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml [hunk 4]
```diff
@@ -242,18 +242,19 @@ objects:
...
+          clusterNetwork:
           - cidr:             10.128.0.0/14
-            hostSubnetLength: 9
+            hostPrefix: 23
           machineCIDR: 10.0.0.0/16
-          serviceCIDR: 172.30.0.0/16
-          type:        OpenShiftSDN
+          serviceNetwork:
+          - 172.30.0.0/16
+          networkType:        OpenShiftSDN
```
Comment (staebler) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: Trim whitespace.

File: ci-operator/templates/openshift/openshift-ansible/cluster-launch-e2e-40.yaml [hunk 5]
```diff
@@ -236,12 +236,13 @@ objects:
...
+          clusterNetwork:
           - cidr:             10.128.0.0/14
-            hostSubnetLength: 9
+            hostPrefix: 23
           machineCIDR: 10.128.0.0/14
-          serviceCIDR: 172.30.0.0/16
-          type:        OpenShiftSDN
+          serviceNetwork:
+          - 172.30.0.0/16
+          networkType:        OpenShiftSDN
```
Comment (staebler) [inline ci-operator/templates/openshift/openshift-ansible/cluster-launch-e2e-40.yaml]: Trim whitespace.

File: ci-operator/templates/openshift/openshift-ansible/cluster-launch-e2e-40.yaml [hunk 6]
```diff
@@ -264,12 +265,13 @@ objects:
...
-          clusterNetworks:
+          clusterNetwork:
           - cidr:             10.128.0.0/14
             hostSubnetLength: 9
           machineCIDR: 10.0.0.0/16
-          serviceCIDR: 172.30.0.0/16
-          type:        OpenShiftSDN
+          serviceNetwork:
+          - 172.30.0.0/16
+          networkType:        OpenShiftSDN
```
Comment (staebler) [inline ci-operator/templates/openshift/openshift-ansible/cluster-launch-e2e-40.yaml]: Trim whitespace.

File: ci-operator/templates/openshift/openshift-ansible/cluster-scaleup-e2e-40.yaml [hunk 7]
```diff
@@ -312,18 +312,19 @@ objects:
...
+          clusterNetwork:
           - cidr:             10.128.0.0/14
-            hostSubnetLength: 9
+            hostPrefix: 23
           machineCIDR: 10.0.0.0/16
-          serviceCIDR: 172.30.0.0/16
-          type:        OpenShiftSDN
+          serviceNetwork:
+          - 172.30.0.0/16
+          networkType:        OpenShiftSDN
```
Comment (staebler) [inline ci-operator/templates/openshift/openshift-ansible/cluster-scaleup-e2e-40.yaml]: Trim whitespace.

File: ci-operator/templates/openshift/openshift-ansible/cluster-scaleup-e2e-40.yaml [hunk 8]
```diff
@@ -336,17 +337,18 @@ objects:
...
-          clusterNetworks:
+          clusterNetwork:
           - cidr:             10.128.0.0/14
-            hostSubnetLength: 9
+            hostPrefix: 23
           machineCIDR: 10.0.0.0/16
-          serviceCIDR: 172.30.0.0/16
+          serviceNetwork:
+          - 172.30.0.0/16
           type:        OpenShiftSDN
```
Comment (staebler) [inline ci-operator/templates/openshift/openshift-ansible/cluster-scaleup-e2e-40.yaml]: s/type/networkType/

Trim whitespace.

**Comment by squeed** [comment]
@staebler thanks for the very detailed review! Fixed.

**Pull request merged**